admin/javascript/src/sections/campaign-admin/controller.js

```diff
@@ -161,7 +162,8 @@ CampaignAdminContainer.propTypes = {
 
 function mapStateToProps(state, ownProps) {
   return {
-    config: state.config.sections[ownProps.sectionConfigKey],
+    config: state.config,
+    sectionConfig: state.config.sections[ownProps.sectionConfigKey],
```

Comment:
Should have a `propTypes` key.

`sectionConfig: React.PropTypes.object.isRequired,`